core/src/test/java/com/novoda/accessibility/ClosedCaptionManagerTest.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.accessibility;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.view.accessibility.CaptioningManager;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@TargetApi(Build.VERSION_CODES.KITKAT)
+public class ClosedCaptionManagerTest {
+
+    @Mock
+    CaptioningManager mockCaptioningManager;
+
+    private ClosedCaptionManager closedCaptionManager;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        closedCaptionManager = new ClosedCaptionManager(mockCaptioningManager);
+    }
+
+    @Test
+    public void whenCheckingIfCloseCaptioningEnabled_thenCaptioningManagerIsCalled() {
```

Comment:
in my opinion adding test when you set what mock needs to return and then checking if it actually returns it is not a valid test at all. Verifying if the method has been called is sufficient enough. 

Although adding two extra tests in this situation is super simple, so we can add them if you still think they're really needed.